classes/Sensors/Menus.php

```diff
@@ -396,10 +422,31 @@ private function EventChangeName($old_menu_name, $new_menu_name)
         ));
     }
 
-    private function EventChangeOrder($menu_name)
+    private function EventChangeOrder($item_name, $menu_name)
     {
         $this->plugin->alerts->Trigger(2085, array(
+            'ItemName' => $item_name,
             'MenuName' => $menu_name
         ));
     }
+
+    private function EventChangeSubItem($item_name, $parent_name)
+    {
+        $this->plugin->alerts->Trigger(2089, array(
+            'ItemName' => $item_name,
+            'ParentName' => $parent_name
+        ));
+    }
+
+    private function GetItemName($term_id, $item_id)
+    {
+        $item_name = '';
+        $menu_items = wp_get_nav_menu_items($term_id);
+        foreach ($menu_items as $menu_item) {
+            if ($menu_item->ID == $item_id) {
+                $item_name = $menu_item->title;
```

Comment:
@carmelopreeostudios would be better to add a `break;` here - once we found the menu title we can stop searching